conda-forge.yml

```diff
@@ -10,3 +10,4 @@ provider:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: azure}
```

Comment:
Add `{linux_aarch64: default, linux_ppc64le: default}` to get native ppc64le builds with travis